### ISSUE TITLE
Allow audio files to display subtitles from one or two SRT files

### ISSUE THREAD

**Issue by vivadavid** [smplayer-dev/smplayer#1101]
**Is your feature request related to a problem? Please describe.**

Hi!

I've found out today about SMPlayer as one the few programmes that let you load two sets of subtitles at the same time for the same video file, which is extremely useful to me. Unfortunately, I can't find a way to load subtitles for an audio file and I was wondering if this tool could be implemented.

I need this feature because sometimes, with Whisper, I turn audio into text and save it as an SRT file. I was using mpv to display the subtitles while playing my audio, but it doesn't allow me to load two sets of subtitles at the same time.

So, to sum up, what I'd need is to play an audio file while displaying one or two sets of subtitles from SRT files.

Thanks in advance!

**Comment by vivadavid** [comment]
Hi!

This is, in fact, an old issue. And I never got a reply, by the way: is there a chance this feature might be implemented in the future?

Thanks!
